packages/generator/src/gulpfileHelpers.ts

```diff
@@ -0,0 +1,97 @@
+/*
```

Comment:
Since this is a gulpfile helper file, would it make more sense to put it with the gulpfiles?

Reply:
I was confused about this one. I decided against it because currently all our code is inside `src` including the other helper class (templateHelpers) we have.